src/DssSpell.t.sol

```diff
@@ -225,12 +226,11 @@ contract DssSpellTest is DSTest, DSMath {
         // Test for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:                 address(0x961eEE551d0Fe5FF8B88db5441875564ac967f09),        // populate with deployed spell if deployed
+            deployed_spell:                 address(0),        // populate with deployed spell if deployed
             deployed_spell_created:         1619196280,        // use get-created-timestamp.sh if deployed
-            previous_spell:                 address(0xDb0D1af4531F59E4E7EfA4ec0AcADec7518D42B6),        // supply if there is a need to test prior to its cast() function being called on-chain.
-            previous_spell_execution_time:  1619372356,        // Time to warp to in order to allow the previous spell to be cast ignored if PREV_SPELL is SpellLike(address(0)).
```

Comment:
I think we should keep this line.
What do you think @brianmcmichael?

Reply:
yes, please leave this here.

Reply:
Actually, I see with you did with the `eta`, that looks great.

Reply:
Yeah we have removed it as it wasn't actually necessary, excepting we are missing some edge case, but doesn't seem to be the case.